### ISSUE TITLE
Use tag:value instead of RDF for the value of referenceType in JSON/YAML

### ISSUE THREAD

**Issue by tsteenbe** [spdx/spdx-spec#451]
See https://github.com/spdx/spdx-spec/blob/development/v2.2.1/examples/SPDXYAMLExample-2.2.spdx.yaml#L177 propose to  make below change so this field aligns with other field where we do not use RDF but tag:value. 

```
referenceType: "http://spdx.org/rdf/references/cpe23Type"
```

to

```
referenceType: "cpe23Type"
```

**Comment by goneall** [comment]
Agreed - I'll update this on the next round of example updates.

**Comment by htttd35** [comment]
❤️

**Comment by goneall** [comment]
@tsteenbe I just updated the examples in PR #519 which should resolve this.

**Comment by goneall** [comment]
Resolved by PR #519
